core/appeal/service.go

```diff
@@ -254,14 +249,14 @@ func (s *Service) Create(ctx context.Context, appeals []*domain.Appeal, opts ...
 			return fmt.Errorf("retrieving creator details: %w", err)
 		}
 
-		if err := s.fillApprovals(appeal, policy); err != nil {
+		if err := appeal.InitApprovals(policy); err != nil {
 			return fmt.Errorf("populating approvals: %w", err)
 		}
 
 		appeal.Init(policy)
 
 		appeal.Policy = policy
```

Comment:
Can these 3 be merged together in this flow? @haveiss @rahmatrhd 
I assume `appeal.Init` should init all.

Reply:
https://github.com/odpf/guardian/pull/322#discussion_r1024852918

we can group this into an appeal method called `ApplyPolicy` or something similar. Because these logic mostly about initializing the appeal based on the policy